frontend/javascripts/oxalis/view/layouting/flex_layout_wrapper.js

```diff
@@ -336,6 +335,7 @@ class FlexLayoutWrapper extends React.PureComponent<Props, State> {
       // Only adjust the internal state if the toggle is not automated and no tab is maximized.
       this.borderOpenStatusWhenNotMaximized[side] = !this.borderOpenStatusWhenNotMaximized[side];
     }
+    this.state.model.doAction(FlexLayout.Actions.selectTab(`${side}-border-tab-container`));
```

Comment:
Performing this action triggers a re-render (on slower system this is probably delayed which is why the bug didn't occur). While re-rendering the border status from the store is used (which is updated in the lines above). Therefore, moving this action made sense to me.

Reply:
If the newest fix (see below) fixes this bug, this change is not necessary anymore. But I think we can just keep it.